File: WorkManagement/ImagesBackEnd-master/ImagesBackEnd-master/Mastex/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Mastex.Models
{
    public class Employee
    {
        public int ID { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string BirthDate { get; set; }
        public string Gender { get; set; }

        public int RankId { get; set; }
        public Rank Rank { get; set; }

        public ICollection<SpreadSheet> SpreadSheet { get; set; }
    }
}

File: WorkManagement/EmployeeUI-master/src/app/_Models/Employee.ts
import { Rank } from './Rank';
import { MyDate } from './MyDate';
import { SpreadSheet } from './SpreadSheet';

export class Employee {
    ID: number;
    FirstName: string;
    Lastname: string;
    BirthDate: string;
    Gender: string;

    RankId: number;
    Rank: Rank;

    MyDate: MyDate;

    SpreadSheet: Array<SpreadSheet>;
}